src/mesh/generic/Boundary.cpp

```diff
@@ -11,6 +11,9 @@
 #include <range/v3/view/zip.hpp>
 
 #include <json/value.h>
+#include <math.h>
```

Comment:
Anything with a `.h` is a C library (usually).  The header you likely want is `<cmath>`.  You can remove the commented stuff out too.

Reply:
thanks! done!